install/installer/pkg/components/ide-service/configmap.go

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+package ide_service
+
+import (
+	"fmt"
+
+	"github.com/gitpod-io/gitpod/common-go/baseserver"
+	"github.com/gitpod-io/gitpod/ide-service-api/config"
+	"github.com/gitpod-io/gitpod/installer/pkg/common"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+func configmap(ctx *common.RenderContext) ([]runtime.Object, error) {
+
+	cfg := config.ServiceConfiguration{
+		Server: &baseserver.Configuration{
+			Services: baseserver.ServicesConfiguration{
+				GRPC: &baseserver.ServerConfiguration{
+					Address: fmt.Sprintf(":%d", GRPCServicePort),
```

Comment:
/hold this will break the new HTTP_PROXY

```suggestion
					Address: fmt.Sprintf("0.0.0.0:%d", GRPCServicePort),
```

Reply:
Addressed

Reply:
Thank you